src/main/kotlin/no/nav/familie/ef/sak/økonomi/StønadtypeExtension.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.økonomi
+
+import no.nav.familie.ef.sak.repository.domain.Stønadstype
+
+fun Stønadstype.tilKlassifisering() = when (this) {
+    Stønadstype.OVERGANGSSTØNAD -> "EFOG"
```

Comment:
Er det mulig å bruke enums i stedet for string?

Reply:
Muligt ja ;) Men blir ett ekstra nivå av mapping, i.e Stønadstype -> KlassifiseringEnum ->ToString utan att jag ser en gevinst? Er det inte bedre behålla som der idag ? :)

Reply:
Ser bruket nå, er enig. Tenker man burde endre KjedeId til å bruke enum-verdiet iaf, fordelen med det er att det er enklere å skjønne hva "klassifisering" er :)